client/themes/morning.css

```diff
@@ -51,43 +51,113 @@
 	color: #b0bacf;
 }
 
-#chat.colored-nicks .user.color-1 { color: #f7adf7; }
-#chat.colored-nicks .user.color-2 { color: #abf99f; }
-#chat.colored-nicks .user.color-3 { color: #86efdc; }
-#chat.colored-nicks .user.color-4 { color: #c890eb; }
-#chat.colored-nicks .user.color-5 { color: #f9a4b3; }
-#chat.colored-nicks .user.color-6 { color: #f7999a; }
-#chat.colored-nicks .user.color-7 { color: #f497b9; }
-#chat.colored-nicks .user.color-8 { color: #f9a9d7; }
-#chat.colored-nicks .user.color-9 { color: #85a7e3; }
-#chat.colored-nicks .user.color-10 { color: #a8b8ff; }
-#chat.colored-nicks .user.color-11 { color: #b695fc; }
-#chat.colored-nicks .user.color-12 { color: #f4aead; }
-#chat.colored-nicks .user.color-13 { color: #fc7cb1; }
-#chat.colored-nicks .user.color-14 { color: #ff72e0; }
-#chat.colored-nicks .user.color-15 { color: #8cb6ea; }
-#chat.colored-nicks .user.color-16 { color: #f9857c; }
-#chat.colored-nicks .user.color-17 { color: #ed9b82; }
-#chat.colored-nicks .user.color-18 { color: #8df484; }
-#chat.colored-nicks .user.color-19 { color: #ffcce3; }
-#chat.colored-nicks .user.color-20 { color: #efcc81; }
-#chat.colored-nicks .user.color-21 { color: #92a2ed; }
-#chat.colored-nicks .user.color-22 { color: #f4d484; }
-#chat.colored-nicks .user.color-23 { color: #97ea70; }
-#chat.colored-nicks .user.color-24 { color: #fcbbba; }
-#chat.colored-nicks .user.color-25 { color: #eef975; }
-#chat.colored-nicks .user.color-26 { color: #c7ff93; }
-#chat.colored-nicks .user.color-27 { color: #ffade1; }
-#chat.colored-nicks .user.color-28 { color: #98ecf2; }
-#chat.colored-nicks .user.color-29 { color: #91a2f5; }
-#chat.colored-nicks .user.color-30 { color: #b19aea; }
-#chat.colored-nicks .user.color-31 { color: #f2a4eb; }
-#chat.colored-nicks .user.color-32 { color: #85f27d; }
+#chat.colored-nicks .user.color-1 {
```

Comment:
I think this is where I'm not a huge fan of this. I think we can go too far into the "just use sensible defaults" and end up in a situation where something becomes less readable.

In general, we want newlines in this sort of situation, but in this exact situation, it is definitely better in the before. 

I feel like it's not simple enough to just go "Here are the rules and they are implmeneted", you know?

Reply:
I agree with you here, it's not ideal.
Some of the things it formatted around FontAwesome icon definitions is kind of a mess before, but I agree that for those long lists of colors like these ones, we're better off without those changes.

There is no way to ignore portions of files (and even then, it's not rule-based so we'd ignore all of it), and overall I find it not super useful on our CSS files which are pretty decent already. I'll remove CSS formatting from the PR.